### ISSUE TITLE
Recreate the repository as a fork from the original library.

### ISSUE THREAD

**Issue by Avamander** [marcoschwartz/LiquidCrystal_I2C#4]
As this library is actually a fork, could you please fork the original library and recreate the changes made.

**Comment by marcoschwartz** [comment]
I agree with you, but I fear this would mess up things with the Arduino library manager if there are two versions of this library. Do you know more about this ?

**Comment by Avamander** [comment]
If you delete this repository and then recreate it as a fork with the same
name and everything and then tell the Arduino maintainers to update their
database or whatever then it should be fine.

**Comment by mcauser** [comment]
@Avamander what is the original library url?

**Comment by tablatronix** [comment]
:+1:
